packages/svelte2tsx/src/htmlxtojsx/utils/node-utils.ts

```diff
@@ -273,3 +273,36 @@ export function buildTemplateString(
         str.appendLeft(attr.end, trailingOverride);
     }
 }
+
+/*
+ * Found if there's a member access trailing behind an expression
+ * Usually it's there because of the preprocess we have before svelte parses
+ * the template
+ */
+export function getNodeEndIncludingTrailingPropertyAccess(
+    originalText: string,
+    position: number
+): number {
+    let index = position;
+
+    while (index < originalText.length) {
```

Comment:
Could we simplify this to `originalText.indexOf('}', position) - 1` ? This is always used in a situation where there's a closing bracket of a mustache tag at the end. Or is this wrong because we only can do this for property assignments and not things like `+`, `&` etc which we also blank?
(having written this out, I think so, so I think it's fine as it is now)

Reply:
Yeah, it's currently only used for `.` and `?.`. Not other operators we blanked. This condition for the while loop is mostly for preventing an infinite loop. In most cases, it would hit other characters and `break` the loop.